### PR TITLE
feat!: new & accessible colours

### PR DESCRIPTION
**Pull request by dozortsev**
## 🖼 Context

We recently updated [Neptune colors](https://neptune.wise.com/design/colour/) to make them more accessible, plus introduced changes into our tokens naming to make them semantic (https://github.com/transferwise/neptune-tokens/pull/19).

## 🚀 Changes

- **BREAKING CHANGE**: updated `@transferwise/neptune-tokens` package to latest which brings different (new) set of CSS custom properties/LESS variables
- use new colors as CSS custom properties instead of old colors being used as LESS variables.
- updated docs:
      - [Working with styles](https://transferwise.github.io/neptune-web/branch/accessible-colours/about/Styles) - added more info about PostCSS setup.
      - [Tokens | Colors](https://transferwise.github.io/neptune-web/branch/accessible-colours/styles/tokens/Colors) - updated list of new color tokens
      - [Addons | Background utilities](https://transferwise.github.io/neptune-web/branch/accessible-colours/styles/addons/BackgroundUtilities) - updated list of `.bg-*` utility classes
- copied over content of some styles from Bootstrap to be able to adjust colors in them

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/css/src/less/addons/_background-utilities.less [hunk 1]
```diff
@@ -2,25 +2,32 @@
 
 // Contextual backgrounds
 
+// @DEPRECATED: .bg-default use .bg-neutral instead
+.bg-neutral,
```
Comment (dozortsev) [inline packages/css/src/less/addons/_background-utilities.less:6]: see these docs about this utility classes here https://transferwise.github.io/neptune-web/branch/accessible-colours/styles/addons/BackgroundUtilities

File: packages/css/src/less/badge.less [hunk 2]
```diff
@@ -1,5 +1,78 @@
 // TODO: remove bootstrap dependencies
-
-// variables
 @import (reference) '../variables/legacy-variables.less';
-@import 'bootstrap/less/badges.less';
+
+// Base class
+.badge {
```
Comment (dozortsev) [inline packages/css/src/less/badge.less:5]: copied from Bootstrap

File: packages/css/src/less/breadcrumbs.less [hunk 3]
```diff
@@ -2,4 +2,28 @@
 
 // variables
 @import (reference) '../variables/legacy-variables.less';
-@import 'bootstrap/less/breadcrumbs.less';
+
```
Comment (dozortsev) [inline packages/css/src/less/breadcrumbs.less:5]: copied from Bootstrap in order to adjust colors

File: packages/css/src/less/close.less [hunk 4]
```diff
@@ -5,4 +5,42 @@
 
 // mixins
 @import (reference) 'bootstrap/less/mixins/tab-focus.less';
-@import 'bootstrap/less/close.less';
+
+//
+// Close icons
+// --------------------------------------------------
+.close {
```
Comment (dozortsev) [inline packages/css/src/less/close.less:12]: copied from Bootstrap in order to adjust colors

File: packages/css/src/less/dropdowns.less [hunk 5]
```diff
@@ -0,0 +1,355 @@
+@import (reference) '../variables/legacy-variables.less';
```
Comment (dozortsev) [inline packages/css/src/less/dropdowns.less:1]: copied from Bootstrap in order to adjust colors

File: packages/docs/pages/about/Styles.mdx [hunk 6]
```diff
@@ -1,6 +1,8 @@
+import { Link } from '@transferwise/components';
+
 import { DocumentLink } from '../../utils/pageUtils';
```
Comment (dozortsev) [inline packages/docs/pages/about/Styles.mdx:4]: rendered https://transferwise.github.io/neptune-web/branch/accessible-colours/about/Styles

File: packages/docs/pages/styles/addons/BackgroundUtilities.mdx [hunk 7]
```diff
@@ -1,19 +1,24 @@
-export function BackgroundToken(props) {
+import { Link } from '@transferwise/components';
```
Comment (dozortsev) [inline packages/docs/pages/styles/addons/BackgroundUtilities.mdx:1]: rendered https://transferwise.github.io/neptune-web/branch/accessible-colours/styles/addons/BackgroundUtilities

File: packages/docs/pages/styles/tokens/Colors.tsx [hunk 8]
```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/function-component-definition */
+import { Link } from '@transferwise/components';
```
Comment (dozortsev) [inline packages/docs/pages/styles/tokens/Colors.tsx:2]: rendered https://transferwise.github.io/neptune-web/branch/accessible-colours/styles/tokens/Colors

File: packages/docs/public/static/assets/main.css [hunk 9]
```diff
@@ -20,7 +20,9 @@ p + h2[id] {
 
 .Header {
   min-height: 100vh;
-  background: var(--color-primary);
+
+  /* TODO: fix this while theming project */
+  background: var(--color-base-navy-mid);
```
Comment (gusso) [inline packages/docs/public/static/assets/main.css:25]:
```suggestion
  background: var(--color-base-navy-light);
```

File: packages/docs/public/static/assets/main.css [hunk 10]
```diff
@@ -108,7 +110,7 @@ p + h2[id] {
 .Nav__Group {
   padding: var(--size-16) var(--size-16) var(--size-8);
   text-transform: uppercase;
-  color: @color-base-grey-light;
+  color: var(--color-content-secondary);
```
Comment (gusso) [inline packages/docs/public/static/assets/main.css:113]:
```suggestion
  color: var(--color-base-smoke-mid);
```

File: packages/docs/public/static/assets/main.css [hunk 11]
```diff
@@ -20,7 +20,9 @@ p + h2[id] {
```
Comment (gusso) [inline packages/docs/public/static/assets/main.css:20]: This background colour is causing 2 background neutrals to be overlapped and making the sidebar darker, we can probably remove this one.

File: packages/components/src/chip/Chip.less [hunk 12]
```diff
@@ -1,14 +1,14 @@
 @import (reference) '@transferwise/neptune-css/src/variables/neptune-tokens.less';
 
 .chip {
-  color: var(--color-text-primary);
-  background: @color-base-smoke-light;
+  color: var(--color-content-primary);
+  background: var(--color-interactive-secondary);
```
Comment (gusso) [inline packages/components/src/chip/Chip.less:5]:
```suggestion
  background: var(--color-background-neutral);
```

**Review by gusso — CHANGES_REQUESTED** [review]
Almost totally perfect, just a couple of changes I suggested ⚡️

File: packages/docs/public/static/assets/main.css (hunk 11 above)
Comment (dozortsev) [reply packages/docs/public/static/assets/main.css:20]: 🆗

**Review by gusso — APPROVED** [review]

**Pull request merged**